### ISSUE TITLE
nested dir aliases show no icon

### ISSUE THREAD

**Issue by bugabinga** [reujab/silver#7]
Given the following config (fish):
```fish
set SILVER status:black:white dir:blue:black git:green:black cmdtime:magenta:black
set -x SILVER_DIR_LENGTH 1
set -x SILVER_DIR_ALIASES $HOME/Workspace:\ue796:$HOME/Workspace/go:\ue627
set -x SILVER_SHELL fish
eval (silver init)
```
I expected to see the go icon when navigating to `~/Workspace/go`.
But the prompt shows: ` /go  1ms `

I am unsure if the output should be `<workspace icon>/<go icon>` or simply `<go icon>`.

> Note: The home directory seems to be a special case here...

**Comment by reujab** [comment]
Fixed in [v1.0.6](https://github.com/reujab/silver/releases). Thanks for reporting this.

**Comment by bugabinga** [comment]
I thank you!
